src/main/java/org/reactome/qa/doisuggester/Main.java

```diff
@@ -50,213 +56,201 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 			Map<GKInstance, Set<Map<GKInstance, GKInstance>>> pathway2Reactions2NewIEs = new HashMap<>();
 
 			// Iterate through all ReactionlikeEvents (RlEs) in the 'current' test_reactome
-			for (GKInstance currentRLE : (Collection<GKInstance>) dbAdaptor.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent))
+			Collection<GKInstance> RLEs = (Collection<GKInstance>) dbAdaptor.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent);
+			System.out.println(RLEs.size() + " Reaction-like events will be checked.");
```

Comment:
For the System.out.println statements, describing the program's progress would it be better to replace them with a logger?

Reply:
It runs in about 20 seconds so I don't think progress output is necessary.